.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         node-version: [10, 12, 14, 16]
     services:
       redis:
-        image: redis
+        image: redis:5
```

Comment:
why it is not the latest? if validating the error message is not possible or hard to maintain. change the test assertion to `t.ok(err)` is fine (it means error must exist).

Reply:
I kinda disagree this time, I think that relying on the `latest` tag of an image is not so good, because in the next major they can totally change their APIs and i dont expect the whole CI to fail due to this, i would update the version incrementally when needed and test the behavior with the new major releases. 
Futhermore i would expect to work with the same version between the local docker environment and the CI one.

I had bad episodes with `latest` tag, it was really hard to debug when something was silently failing. Does this make sense to you? Or still prefering the `latest` tag?

Reply:
The document stated that we are required for `redis@5` or greater and the CI is failing for `redis@latest`. I am fine for a fixed version, but I think the test should pass for both `redis@5` and `redis@6` currently.

Reply:
Thats a really nice point :+1: How would u tackle this? testing it locally with different versions or implementing inside the CI?
Im fine to revert back to lastest tho, but i had so many ghosts with it :smiley: 

BTW I agree on generalizing the error inside the tests, uploading a patch soon. Thanks for the tips.

Reply:
I think we can make use of the `strategy.matrix`, technically it may pass to `services` like `steps` below. But I haven't tested it.

e.g.
```
strategy:
      matrix:
        node-version: [10, 12, 14, 16]
        redis-version: [5, 6]
    services:
      redis:
        image: redis:${{ matrix.redis-version }}
```

Reply:
Yeah came up with the same idea, testing a CI to see if it feasible.